app/scripts/views/coppa/coppa-age-input.js

```diff
@@ -56,6 +56,22 @@ define(function (require, exports, module) {
       age.val(age.val().substr(0, 3));
     },
 
+    onKeyDown: function (event) {
+      // helper function to check for digit
+      function isKeyADigit (keyCode) {
+        return ((keyCode == 8) || // backspace
```

Comment:
Here's an straight forward explanation of how type coercion works in JS - https://stackoverflow.com/questions/19915688/what-exactly-is-type-coercion-in-javascript